llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -1001,6 +1001,53 @@ RISCVTTIImpl::getMinMaxReductionCost(Intrinsic::ID IID, VectorType *Ty,
       return getArithmeticReductionCost(Instruction::And, Ty, FMF, CostKind);
   }
 
+  if (IID == Intrinsic::maximum || IID == Intrinsic::minimum) {
+    SmallVector<unsigned, 5> SplitOps;
```

Comment:
Can you subset this patch to the case where LT.first == 1?  The splitting logic still looks too conservative here, but let's separate that discussion off.